Include.psm1

```diff
@@ -691,15 +689,14 @@ function Get-ChildItemContent {
 filter ConvertTo-Hash { 
     [CmdletBinding()]
     $Hash = $_
-    switch ([math]::truncate([math]::log($Hash, [Math]::Pow(1000, 1)))) { 
-        $null { "0  H" }
-        "-Infinity" { "0  H" }
+    switch ([math]::Abs([math]::truncate([math]::log($Hash, [Math]::Pow(1000, 1))))) { 
```

Comment:
Why are you converting negative numbers to positive?

Reply:
Also, what bug have you found in 'ConvertTo-Hash'? Have you got an input and expected output example?